depends/libhdfs3/osx/README.md

```diff
@@ -0,0 +1,14 @@
+# Packging LibHdfs3 for Mac OS X
+This directory contains 
+- build script for osx for developing libhdfs3
+- homebrew formula to install libhdfs3 on osx
+
+## Installing libhdfs3
+All you need to do is run
+
+```bash
+./install_homebrew_pkg.sh
+```
+
+What that will do is setup a local tap for immuta and then "install" our
```

Comment:
Before I submit this back. I will probably remove the examples directory. This is just for you Kyle.